src/main/java/seedu/address/model/recipe/Recipe.java

```diff
@@ -45,11 +44,9 @@ public Recipe(Name name, CompletionTime completionTime, ServingSize servingSize,
         this.ingredients.addAll(ingredients);
         this.steps.addAll(steps);
         this.tags.addAll(tags);
-        initSearchSet();
+        this.initSearchSet();
```

Comment:
Nice was wondering if i should suggest adding the "this" call since we do it for all private variables/fn calls - now everything's consistent yay